sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 7.0.0-preview.2 (Unreleased)
 
+- Fixes an issue where non-retryable errors caused by a connection disconnecting were not getting surfaced to the user's registered error handler
```

Comment:
This statement is no longer true because non retryable errors caused by connection disconnecting will be treated as retryable...

Reply:
Good point.
```suggestion
- Any error caused by a connection disconnecting is retried as per the user-provided retry options and the failures would get surfaced to the user's registered error handler
```

How's this?

Reply:
Looks good